Sources/KSCrashRecording/KSCrash.m

```diff
@@ -297,7 +297,6 @@ - (NSDictionary*) systemInfo
     COPY_STRING(kernelVersion);
     COPY_STRING(osVersion);
     COPY_PRIMITIVE(isJailbroken);
-    COPY_STRING(bootTime);
```

Comment:
The tricky part is here. The public API used to reference `bootTime` and `storageSize`, but now these fields are no longer populated, and we cannot reference BootTime and DiscSpace monitors. In my opinion, the most reasonable action is to leave it as it is.

Reply:
So you mean to leave `COPY_STRING(bootTime);`, but it would not be populated with any data?

Reply:
No, something like `// `bootTime` is populated in BootTimeMonitor product/subspec`.
Just to make sure it's clear why these 2 are not populated here.

Reply:
> is populated in BootTimeMonitor product/subspec

But it would mean that the user will get this data with `-[KSCrash systemInfo]` call, but I want to clarify that this is not the case.

Reply:
Ah, right. Then comments like `// bootTime is not populated here. See BootTimeMonitor product/subspec. 
So that in case someone was using this method they would immediately see that this data is no longer available (as there's no way to explicitly highlight this via breaking change to API).